claims_hosp/delphi_claims_hosp/update_indicator.py

```diff
@@ -267,4 +264,4 @@ def write_to_csv(self, output_dict, output_path="./receiving"):
                                 "%s,%f,%s,%s,%s\n" % (geo_id, val, "NA", "NA", "NA"))
                         out_n += 1
 
-        logging.debug("wrote %d rows for %d %s", out_n, len(geo_ids), geo_level)
+        self.logger.debug("wrote %d rows for %d %s", out_n, len(geo_ids), geo_level)
```

Comment:
we will probably never see the `debug` level messages in practice, but we might as well make them follow the convention too:
```suggestion
        self.logger.debug("wrote rows", num_rows=out_n, num_geo_ids=len(geo_ids), geo_levelgeo_level)
```